lock-api/src/main/java/com/palantir/lock/LockService.java

```diff
@@ -213,55 +159,31 @@ LockResponse lockWithFullLockResponse(@Safe @PathParam("client") LockClient clie
      *
      * @throws IllegalArgumentException if {@code grantId} is invalid
      */
-    @POST
-    @Path("use-grant-id/{client: .*}")
-    @Produces(MediaType.APPLICATION_JSON)
-    @Consumes(MediaType.APPLICATION_JSON)
-    @NonIdempotent HeldLocksToken useGrant(@Safe @PathParam("client") LockClient client, BigInteger grantId);
+    HeldLocksToken useGrant(LockClient client, BigInteger grantId);
 
     /**
      * This method is the same as <code>getMinLockedInVersionId(LockClient.ANONYMOUS)</code>.
      * @deprecated use {@link #getMinLockedInVersionId(LockClient)} instead
      */
-    @POST
-    @Path("min-locked-in-version-id")
-    @Produces(MediaType.APPLICATION_JSON)
-    @Consumes(MediaType.APPLICATION_JSON)
     @Deprecated
-    @Idempotent
-    @Nullable Long getMinLockedInVersionId();
+    @Nullable
+    Long getMinLockedInVersionId();
 
     /**
      * Returns the minimum version ID for all locks that are currently acquired
      * (by everyone), or {@code null} if none of these active locks specified a
      * version ID in their {@link LockRequest}s.
      */
-    @POST
-    @Path("min-locked-in-version-id-for-client/{client: .*}")
-    @Produces(MediaType.APPLICATION_JSON)
-    @Consumes(MediaType.APPLICATION_JSON)
-    Long getMinLockedInVersionId(@Safe @PathParam("client") LockClient client);
+    @Nullable
+    Long getMinLockedInVersionId(LockClient client);
 
     /** Returns the options used to configure the lock server. */
-    @POST
-    @Path("lock-server-options")
-    @Produces(MediaType.APPLICATION_JSON)
-    @Consumes(MediaType.APPLICATION_JSON)
-    @Idempotent LockServerOptions getLockServerOptions();
+    LockServerOptions getLockServerOptions();
 
     /** Returns the current time in milliseconds on the server. */
-    @POST
-    @Path("current-time-millis")
-    @Produces(MediaType.APPLICATION_JSON)
-    @Consumes(MediaType.APPLICATION_JSON)
     @Override
-    @Idempotent long currentTimeMillis();
+    long currentTimeMillis();
 
-    @POST
-    @Path("log-current-state")
-    @Produces(MediaType.APPLICATION_JSON)
-    @Consumes(MediaType.APPLICATION_JSON)
     @Override
-    @Idempotent void logCurrentState();
```

Comment:
Can we safely remove this? This might be used by other internal clients.

Reply:
It's a method of RemoteLockService, the @Override does nothing?